src/legend.ts

```diff
@@ -1,15 +1,15 @@
+import {BaseLegendConfig, Legend as VgLegend, LegendConfigNoSignals, LegendDirection, LegendOrient} from 'vega';
 import {DateTime} from './datetime';
 import {Guide, GuideEncodingEntry, VlOnlyGuideConfig} from './guide';
 import {Flag, flagKeys} from './util';
-import {VgLegend, VgLegendBase, VgLegendConfig} from './vega.schema';
 
 
-export interface LegendConfig extends VgLegendConfig, VlOnlyGuideConfig {}
+export interface LegendConfig extends LegendConfigNoSignals, VlOnlyGuideConfig {}
 
 /**
  * Properties of a legend or boolean flag for determining whether to show it.
  */
-export interface Legend extends VgLegendBase, Guide {
+export interface Legend extends BaseLegendConfig, Guide {
```

Comment:
Shouldn't everything down here now should all extend from vega-typings?

A few exception
- `type` -- we want to override "default") 
- `orient` -- (maybe not an exception if the default is not different from Vega)
- `encoding` - VL specific.

Reply:
We cannot extend and change the type of some.